koku/masu/processor/ocp/ocp_cloud_parquet_summary_updater.py

```diff
@@ -119,7 +119,27 @@ def get_infra_map(self, start_date, end_date):
             self.provider_type in Provider.CLOUD_PROVIDER_LIST and self._provider_uuid not in infra_provider_uuids
         ):
             infra_map = self._generate_ocp_infra_map_from_sql(start_date, end_date)
-        return infra_map
+
+        summary_infra_map = {}
+        with OCPReportDBAccessor(self._schema) as accessor:
+            # Only include OCP providers for OCP on Cloud summary if we have a matching report period
+            for ocp_provider_uuid in infra_map:
+                ctx = {
+                    "schema": self._schema,
+                    "ocp_provider_uuid": ocp_provider_uuid,
+                    "provider_uuid": infra_map[ocp_provider_uuid][0],
+                    "provider_type": infra_map[ocp_provider_uuid][1],
+                }
+                if accessor.report_periods_for_provider_uuid(ocp_provider_uuid, start_date):
```

Comment:
Question:
> our large customers they can have 50+ clusters

could this be making multiple calls to the db as we call this func inside the for loop?

Reply:
Correct, its been called once for each distinct OCP source.